src/Features/Core/Portable/ExtractInterface/ExtractInterfaceCodeAction.cs

```diff
@@ -22,12 +22,13 @@ public override object GetOptions(CancellationToken cancellationToken)
             ? string.Empty
             : _typeAnalysisResult.TypeToExtractFrom.ContainingNamespace.ToDisplayString();
 
-        return AbstractExtractInterfaceService.GetExtractInterfaceOptionsAsync(
+        return AbstractExtractInterfaceService.GetExtractInterfaceOptions(
             _typeAnalysisResult.DocumentToExtractFrom,
             _typeAnalysisResult.TypeToExtractFrom,
             _typeAnalysisResult.ExtractableMembers,
             containingNamespaceDisplay,
-            cancellationToken).WaitAndGetResult_CanCallOnBackground(cancellationToken);
```

Comment:
removing this was the goal.  Note: This was terrifying as the impl was doing a JTF switch to teh ui thread.  this was only ok as we're guaranteed to always be on the UI thread to begin with when calling GetOptions (guaranteed as a publicly documented contract of CodeActionWithOptions:

```
    /// <summary>
    /// Gets the options to use with this code action.
    /// This method is guaranteed to be called on the UI thread.
    /// </summary>
```